File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassWithMappedTypes/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassWithMappedTypes.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Entity = "entity";

    public static final String EntityConnection = "entityConnection";
  }

  public static class ENTITY {
    public static final String TYPE_NAME = "Entity";

    public static final String Long = "long";

    public static final String DateTime = "dateTime";
  }

  public static class ENTITYCONNECTION {
    public static final String TYPE_NAME = "EntityConnection";

    public static final String PageInfo = "pageInfo";

    public static final String Edges = "edges";
  }

  public static class ENTITYEDGE {
    public static final String TYPE_NAME = "EntityEdge";

    public static final String Cursor = "cursor";

    public static final String Node = "node";
  }

  public static class PAGEINFO {
    public static final String TYPE_NAME = "PageInfo";

    public static final String StartCursor = "startCursor";

    public static final String EndCursor = "endCursor";

    public static final String HasNextPage = "hasNextPage";

    public static final String HasPreviousPage = "hasPreviousPage";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/dataClassWithMappedTypes/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.dataClassWithMappedTypes.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object ENTITY {
    public const val TYPE_NAME: String = "Entity"

    public const val DateTime: String = "dateTime"

    public const val Long: String = "long"
  }

  public object ENTITYCONNECTION {
    public const val TYPE_NAME: String = "EntityConnection"

    public const val Edges: String = "edges"

    public const val PageInfo: String = "pageInfo"
  }

  public object ENTITYEDGE {
    public const val TYPE_NAME: String = "EntityEdge"

    public const val Cursor: String = "cursor"

    public const val Node: String = "node"
  }

  public object PAGEINFO {
    public const val TYPE_NAME: String = "PageInfo"

    public const val EndCursor: String = "endCursor"

    public const val HasNextPage: String = "hasNextPage"

    public const val HasPreviousPage: String = "hasPreviousPage"

    public const val StartCursor: String = "startCursor"
  }

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Entity: String = "entity"

    public const val EntityConnection: String = "entityConnection"
  }
}
